messaging-experiments.yaml

```diff
@@ -226,3 +226,106 @@
                   label: Not now
                   action:
                     navigate: true
+- id: MULTI-STAGE-ABOUTWELCOME-V1-78-BETA-V2
+  enabled: true
+  filter_expression: env.version >= '78.' && env.version < '79.' && env.channel == 'beta' && env.appinfo.appBuildID >= '20200610093716'
+  targeting: localeLanguageCode == "en" && [userId, "bug-1644521-message-multi-stage-aboutwelcome-v1-78-beta-v2-beta-78-79"]|bucketSample(0, 5000, 10000) &&
```

Comment:
Leaving bucketSample to target 50% of population, we should update experimenter ticket to reflect that